src/fairchem/data/oc/utils/vasp_flags.py

```diff
@@ -82,20 +81,12 @@
     "ncore": 4,
 }
 
-RELAX_FLAGS = {"ibrion": 2, "nsw": 500}
+RELAX_FLAGS = {"ibrion": 2, "nsw": 5}
```

Comment:
curiosity, only 5 ionic steps?

Reply:
yeah we're really leaning into more configurational diversity rather than highly correlated relaxation frames. 5 is rather arbitrary but mainly because the initial steps can often not converge and later ionic steps benefit from a better initial guess